kubernetes/samples/charts/apache-samples/README.md

```diff
@@ -1,8 +1,8 @@
 # Apache load balancer samples
 
-The sample package contains two samples that use the [Apache helm chart](../apache-webtier/README.md). The samples use the Docker image for the Apache HTTP Server with the 12.2.1.3.0 Oracle WebLogic Server Proxy Plugin.  See the details in [Apache HTTP Server with Oracle WebLogic Server Proxy Plugin on Docker](https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-webtier-apache).
+The sample package contains two samples that use the [Apache helm chart](../apache-webtier/README.md). The samples use the Docker image for the Apache HTTP Server with the 12.2.1.3.0 and 12.2.1.4.0 Oracle WebLogic Server Proxy Plugin.  See the details in [Apache HTTP Server with Oracle WebLogic Server Proxy Plugin on Docker](https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-webtier-apache).
 
 * [The default sample](default-sample/README.md) uses the built-in configuration in the above Docker image.
```

Comment:
in the above Docker image. -> in the Docker image.

Reply:
Updated.